gulpfile.js

```diff
@@ -46,10 +51,13 @@ function handleWatchEvent(event) {
 gulp.task('coffee', function() {
   return gulp.src(
       [
+        './app/Resources/assets/coffee/***.coffee',
         './src/**/Resources/assets/coffee/***.coffee',
         './vendor/sumocoders/**/Resources/assets/coffee/***.coffee'
       ]
   )
+      .pipe(plumber())
+      .pipe(gulpif(minify == false, sourcemaps.init()))
```

Comment:
goh, ik had dat eerst gedaan, maar vond `minify == false` vlotter lezen doordat het explicieter is.